collections/build-collections.py

```diff
@@ -5,7 +5,7 @@
 
 
 def write_file(dict, path):
-    manifest_bucket = "manifest-pipeline-v3-manifestbucket-1dxmq1ws0o3ah"
+    manifest_bucket = "marble-manifest-prod-manifestbucket-lpnnaj4jaxl5"
```

Comment:
Can this be parameterized? Or read from the outputs of the prod stack?

Reply:
I currently only look at this as a stop-gap till the pipeline can do the work of creating these.  So I have not put a lot of time into building a reusable system.